spec/workers/future_learn_course_worker_spec.rb

```diff
@@ -0,0 +1,130 @@
+    "language": "en",
+    "hours_per_week": 4,
+    "has_certificates": true,
+    "categories": [
+      "Online & Digital"
+    ],
+    "educator": "Karsten Øster Lundqvist (Lead Educator)",
+    "organisation": {
+      "url": "http://www.reading.ac.uk",
+      "name": "University of Reading"
+    },
+    "trailer": "//view.vzaar.com/4251365/video",
+    "runs": [
+      {
+        "uuid": "6f24cc01-d18f-42f8-b679-ed0830db5d32",
+        "start_date": "2013-10-28",
+        "duration_in_weeks": 7
+      },
+      {
+        "uuid": "2b3b6da3-6da7-4ae2-84fe-9ddf08f91327",
+        "start_date": "2014-02-24",
+        "duration_in_weeks": 7
+      },
+      {
+        "uuid": "ed6a9426-6518-453f-9a13-6e6b1b8779ca",
+        "start_date": "2014-10-20",
+        "duration_in_weeks": 7}]
+  }]'
+  end
+
+  let(:courses_json) { JSON.parse courses_response }
+  let!(:free_course_track_type) { FactoryGirl.create :course_track_type, type_of_achievement: 'nothing' }
+  let!(:certificate_course_track_type) { FactoryGirl.create :certificate_course_track_type, type_of_achievement: 'certificate' }
+
+  it 'delivers MOOCProvider' do
+    expect(future_learn_course_worker.mooc_provider).to eql mooc_provider
+  end
+
+  it 'gets an API response' do
+    expect(future_learn_course_worker.course_data).to_not be_nil
```

Comment:
As done with the last pull review you reviewed, I changed `.to_not` to `.not_to`. Both is possible, but   `.not_to` is preferred according to Ruby conventions.